apparmor.d/io.elementary.lightdm-guest-session

```diff
@@ -0,0 +1,33 @@
+# vim:syntax=apparmor
+# Profile for restricting lightdm guest session
+
+#include <tunables/global>
+
+/usr/lib/lightdm/lightdm-guest-session {
+  # Most applications are confined via the main abstraction
+  #include <abstractions/lightdm>
+
+  # Prevent ICEauthority error in e.g. MATE
+  owner /{,var/}run/user/[0-9]*/ICEauthority* l,
+
+  # chromium-browser needs special confinement due to its sandboxing
+  #include <abstractions/lightdm_chromium-browser>
```

Comment:
Unsure why this is needed in the guest session?

Reply:
Yeah that's a great point. This is copied from the Ubuntu profile but it could be removed for us